File: tutorials/vbnet/loading_balance/pi_i.vb
Public NotInheritable Class piConst

	Private Sub New()
	End Sub

	'defines for service Pi
	Public Const sidPi As UInteger = (SocketProAdapter.BaseServiceID.sidReserved + 5)
	Public Const sidPiWorker As UInteger = sidPi + 1

	Public Const idComputePi As UShort = (CUShort(SocketProAdapter.tagBaseRequestID.idReservedTwo) + 1)
End Class

File: tutorials/csharp/loading_balance/pi_i.cs
public static class piConst
{
	//defines for service Pi
	public const uint sidPi = (SocketProAdapter.BaseServiceID.sidReserved + 5);
    public const uint sidPiWorker = sidPi + 1;

	public const ushort idComputePi = ((ushort)SocketProAdapter.tagBaseRequestID.idReservedTwo + 1);
}